src/PrestaShopBundle/Controller/Admin/Configure/ShopParameters/TitleController.php

```diff
@@ -148,8 +153,55 @@ public function deleteAction(int $titleId): RedirectResponse
      *
      * @return RedirectResponse
      */
-    public function bulkDeleteAction(): RedirectResponse
+    public function bulkDeleteAction(Request $request): RedirectResponse
     {
+        $titleIds = $this->getBulkTitlesFromRequest($request);
+
+        try {
+            $this->getCommandBus()->handle(new BulkDeleteTitleCommand($titleIds));
+            $this->addFlash(
+                'success',
+                $this->trans('Successful deletion', 'Admin.Notifications.Success')
+            );
+        } catch (TitleException $e) {
+            $this->addFlash('error', $this->getErrorMessageForException($e, $this->getErrorMessages($e)));
+        }
+
         return $this->redirectToRoute('admin_title_index');
     }
+
+    /**
+     * @param Request $request
+     *
+     * @return array
+     */
+    private function getBulkTitlesFromRequest(Request $request): array
+    {
+        $titleIds = $request->request->get('title_title_bulk');
+
+        if (!is_array($titleIds)) {
+            return [];
+        }
+
+        foreach ($titleIds as $i => $titleId) {
+            $titleIds[$i] = (int) $titleId;
+        }
+
+        return $titleIds;
+    }
+
+    /**
+     * @param Throwable|null $e
+     *
+     * @return array
+     */
+    private function getErrorMessages(?Throwable $e = null): array
```

Comment:
This argument doesn't seem to be used
```suggestion
    private function getErrorMessages(): array
```

Reply:
Fixed